src/nodes.coffee

```diff
@@ -1252,7 +1252,8 @@ exports.ImportDeclaration = class ImportDeclaration extends ModuleDeclaration
     code.push @makeCode "#{@tab}import "
 
     if @clause? and @clause.length isnt 0
-      for subclause in @clause
+      for subclause, index in @clause
+        code.push @makeCode ', ' if index isnt 0
         code = code.concat subclause.compileNode o
```

Comment:
An earlier version had `firstClause` and `secondClause` but that looked sloppy. I suppose this could technically say `if index is 1` because the index will never hit 2, but that seems unnecessarily opaque.